<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_focos_titulo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>591</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: bold 14pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>ðŸ’¡ InformaciÃ³n y ConfiguraciÃ³n del Sistema de IluminaciÃ³n</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="com">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: bold 16pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_estado">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>190</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listLuminosidad">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>391</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: bold 24pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_conectar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>CONECTAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="puerto_com">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 15px;
color: black;</string>
    </property>
    <property name="text">
     <string>Ingrese el Numero del Puerto COM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_capturar_luz">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>260</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Mostrar Datos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_regresar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2a5298;
color: white;
font: 10pt &quot;Arial&quot;;
border-radius: 10px;
border: 2px solid black;</string>
    </property>
    <property name="text">
     <string>ðŸ”™ Salir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_focos">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffcc00;
color: black;
font: bold 12pt &quot;Arial&quot;;
border: 2px solid black;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string>Leds</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
